app/models/review_turnaround.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: review_turnarounds
+#
+#  id                :bigint           not null, primary key
+#  value             :integer
+#  review_request_id :bigint           not null
+#
+# Indexes
+#
+#  index_review_turnarounds_on_review_request_id  (review_request_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (review_request_id => review_requests.id)
+#
+
+class ReviewTurnaround < ApplicationRecord
+  belongs_to :review_request
+
+  validates :value, presence: true
+  validates :review_request_id, uniqueness: true
+
+  def set_value
```

Comment:
minor comment, but this business logic code could be moved to a service object so to keep models thin

Reply:
Model validations too? Or just the set_value method?

Reply:
just the `set_value`, e.g. instead of calling `set_value` from the other model you could call `ReviewTurnaroundCreator.new(self).call` . what do you think?